_tools/go.mod

```diff
@@ -3,27 +3,219 @@ module github.com/DataDog/dd-trace-go/_tools
 go 1.23.0
 
 require (
+	github.com/golangci/golangci-lint/v2 v2.3.0
 	golang.org/x/perf v0.0.0-20250710210952-7b7c2de18447
-	golang.org/x/tools v0.34.0
+	golang.org/x/tools v0.35.0
 	golang.org/x/vuln v1.1.4
-	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.3.0
-	google.golang.org/protobuf v1.33.0
+	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.5.1
+	google.golang.org/protobuf v1.36.6
 	gotest.tools/gotestsum v1.12.3
```

Comment:
this should be the only place left that controls the versioning for things like `gotestsum`, right? 👍🏼

Reply:
Yes! This should be true for all the tools.
I have some other work that I need to sync when I revive https://github.com/DataDog/dd-trace-go/pull/3727